minecraft_go/software/mc_go/src/touchscreen.c

```diff
@@ -87,5 +67,33 @@ Point touchscreen_get_release(void) {
 	p2.y = (int) (buffer[4]) << 7;
 	p2.y += (int) (buffer[3]);
 
+	p2 = touchscreen_pixel_conversion(p2);
+
 	return p2;
 }
+
+void touchscreen_get_report_packet(unsigned char *buffer, unsigned char touchStatus){

```

Comment:
you'll need to pass the length of the buffer as a parameter. You're extremely lucky here that `sizeof(unsigned char *) = 4`